test/kubernetes/e2e/tests/base/base_suite.go

```diff
@@ -152,7 +152,7 @@ func (s *BaseTestingSuite) BeforeTest(suiteName, testName string) {
 
 	testCase, ok := s.TestCase[testName]
 	if !ok {
-		return
+		s.FailNow("no manifests found for " + testName)
```

Comment:
I think we should allow tests to run without any manifests specified. They could have been deployed during the setup suite as they are common across testss

Reply:
my fear was that tests would silently not apply manifests if you renamed a test and forgot to rename the name in the testCase map (it happened to me)

Reply:
i undid this change